fe/fe-core/src/main/java/org/apache/iceberg/hive/IcebergHiveCatalog.java

```diff
@@ -241,7 +244,7 @@ public void createDb(String dbName, Map<String, String> properties) {
             if (key.equalsIgnoreCase(LOCATION_PROPERTY)) {
                 try {
                     URI uri = new Path(value).toUri();
-                    FileSystem fileSystem = FileSystem.get(uri, new Configuration());
+                    FileSystem fileSystem = FileSystem.get(uri, conf);
```

Comment:
There is another place still use new Configuration, you need replace it.

Reply:
done